README.md

```diff
@@ -721,3 +722,11 @@ the restrictions he needs for the entity.
 
 **IMPORTANT NOTE**: As previously stated, OneOf and AnyOf will behave the same,
 this means that OneOf will work the same way as an AnyOf.
+
+## Properties validation
+Both AsyncAPI and OpenAPI offer the possibility to add properties to limit the
+values that a certain object can take. To validate these properties we annotate
```

Comment:
Maybe instead of saying "limit the values that a certain object can take", saying "apply constraints to the values..." makes it a bit clearer

Reply:
Done